.github/workflows/test.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: .tox
-          key: tox-cache-${{ env.RUN_MATRIX_COMBINATION }}-${{ hashFiles('tox.ini', 'dev-requirements.txt') }}
+          key: v2-build-tox-cache-${{ env.RUN_MATRIX_COMBINATION }}-${{ hashFiles('tox.ini', 'dev-requirements.txt') }}
```

Comment:
What's the rename for?

Reply:
.tox dir was cached and resulted in failed builds. Had to change the key in order to invalidate it.